posix.mak

```diff
@@ -230,6 +230,22 @@ else
  CHANGELOG_VERSION_STABLE := "v${LATEST}..upstream/stable"
 endif
 
+################################################################################
+# Build compression variables
+################################################################################
+# Set to 1 in the command line to minify css and js files
+MINIFY=0
+
+YUICOMPRESSOR_VERSION=2.4.8
+YUICOMPRESSOR=yuicompressor-$(YUICOMPRESSOR_VERSION).jar
+JS_COMPRESSOR_BIN=$G/$(YUICOMPRESSOR)
```

Comment:
That's just confusing, why not call it `YUI_COMPRESSOR_BIN`? It's not clear why there's a rule to download only the JS compression but not the CSS one.